contracts/swapper/LevSwapper/morpho/PendleLevSwapperTemplateMorpho.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.17;
+
+import "./PendleLevSwapperMorpho.sol";
+
+/// @author Angle Labs, Inc.
+/// @notice Template leverage swapper on PT tokens
+contract PendleLevSwapperTemplate is PendleLevSwapperMorpho {
+    constructor(
+        ICoreBorrow _core,
+        IUniswapV3Router _uniV3Router,
+        address _aggregator,
+        IAngleRouterSidechain _angleRouter,
+        IMorphoBase _morpho
+    ) PendleLevSwapperMorpho(_core, _uniV3Router, _aggregator, _angleRouter, _morpho) {}
+
+    /// @inheritdoc BaseLevSwapper
+    function angleStaker() public pure override returns (IBorrowStaker) {
```

Comment:
Can put the empty angleStaker() address directly at the level of the PendleLevSwapperMorpho I guess